.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gutenberg"]
 	path = gutenberg
-	url = ../../cameronvoell/gutenberg.git
+	url = ../../WordPress/gutenberg.git
```

Comment:
thanks for updating this!

Reply:
The failing CI jobs helped me remember it 😆